extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -183,7 +217,111 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._dialogObject.content = [tab];
 	}
 
+	private createResouceButtonsContainer(): azdata.ListViewComponent {
+		const tags = this.getAllResourceTags(this._resourceTypes);
+		if (!tags.includes('All')) {
```

Comment:
I am not sure whether the tags needs to be localized, I would think so, but please confirm with PMs, if so, you need to handle the tag and the tag displayName properly.